mods/crafting/init.lua

```diff
@@ -147,5 +147,5 @@ local removing = {
 api.make_activated_multiblock(recipe, removing)
 
 api.recipes.add("crafting_table_progressive",
-		{"ws_core:wood", "ws_core:wood", [4] = "ws_core:wood", [5] = "ws_core:wood", [10] = "crafting:crafting_table"},
+		{"ws_core:planks_dead", "ws_core:planks_dead", [4] = "ws_core:planks_dead", [5] = "ws_core:planks_dead", [10] = "crafting:crafting_table"},

```

Comment:
Should use groups, I'll try to fix that today and notify you if that works (this is fine for now tho)

Reply:
groups in progresive table didn't work when I tried it

Reply:
I'll fix that later, you can push that now, sorry for requesting changes here